proposals/gc/MVP.md

```diff
@@ -779,6 +779,14 @@ The opcode for heap types is encoded as an `s33`.
 | 0xfb70 | `extern.internalize` | |
 | 0xfb71 | `extern.externalize` | |
 
+## Implementation-defined Limits
```

Comment:
Since these are constraints of the JS API spec, it should make more sense to move this section down below and turn it into a subsection of that one.